site/static/docs/5.3/examples/timeline.html

```diff
@@ -0,0 +1,61 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport"
+        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Timeline Example</title>
+  <link href="https://fonts.googleapis.com/css2?family=Material+Icons" rel="stylesheet">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/5.3.0/css/bootstrap.min.css">
+  <link rel="stylesheet" href="../dist/css/custom.css">
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/prism/1.23.0/themes/prism.min.css" rel="stylesheet" />
+</head>
+<body class="container" style="background-color: #EFEFEF;">
+
+<div class="center-content">
+  <div class="timeline-custom">
+    <div class="timeline-item-custom">
+      <div class="timeline-icon-custom timeline-icon-top"></div>
+      <div class="timeline-content-custom">
+        <h4 class="timeline-title-custom">Fase de inscrições</h4>
+        <p class="timeline-description-custom">de 14/08/2024 a 20/08/2024 às 21:27</p>
+      </div>
+    </div>
+
+    <div class="timeline-item-custom">
+      <div class="timeline-icon-custom timeline-icon-bottom"></div>
+      <div class="timeline-content-custom">
+        <h4 class="timeline-title-custom-end">Publicação final do resultado</h4>
+      </div>
+    </div>
+  </div>
+
+  <br><br><br>
+
+  <div class="box">
+    <pre><code class="language-html">
+      &lt;div class="timeline-custom"&gt;
```

Comment:
```suggestion
      &lt;link rel="stylesheet" href="../dist/css/custom.css"&gt;
      &lt;div class="timeline-custom"&gt;
```
apenas para complementar a documentação do componente para que quando um usuário novo for usar ele saber que precisará importar o arquivo `custom.css`